pkg/credentials/aws.go

```diff
@@ -31,3 +42,28 @@ func (r *AWSCredentialsResponse) String() string {
 func (r *AWSCredentialsResponse) FmtExport() string {
 	return fmt.Sprintf(AwsExportFormat, r.AccessKeyID, r.SecretAccessKey, r.SessionToken, r.Region)
 }
+
+// AWSV2Config returns an aws-sdk-go-v2 config that can be used to programmatically access the AWS API
+func (r *AWSCredentialsResponse) AWSV2Config() (aws.Config, error) {
+	bpConfig, err := bpconfig.GetBackplaneConfiguration()
+	if err != nil {
+		return aws.Config{}, fmt.Errorf("failed to load backplane config file: %w", err)
+	}
+
+	proxyURL, err := url.Parse(bpConfig.ProxyURL)
+	if err != nil {
+		return aws.Config{}, fmt.Errorf("failed to parse proxy_url from backplane config file: %w", err)
+	}
```

Comment:
This may already checked when login to the cluster 
https://github.com/openshift/backplane-cli/blob/23570ca9b085a00e17ed5eb143fe31178cfb8b89/pkg/cli/config/config.go#L99

Reply:
Yeah, but this function will not be called after `ocm backplane login` - e.g. https://github.com/openshift/osdctl/blob/1b906de67017e15057a6312d698ac28565250941/pkg/osdCloud/aws.go#L182-L190

Though in general, `ocm backplane cloud *` can be called without needing to `ocm backplane login`

Reply:
ahh got your point, We should move all config validation to `pkg/cli/config` rather than individual sub-commands .. I'll create a card for moving that. I'm fine to keep it here for now.

Reply:
thanks @samanthajayasinghe , let's create a card for tracking this.